lib/events/food-events.js

```diff
@@ -1,8 +1,22 @@
-const Food = require('../models/Food')
+const Food   = require('../models/Food')
+const $submit = $('#add-food')
 
 $(document).ready(() => {
   Food.allFoodToHTML()
     .then(function(foodHTML) {
       $('#foods-table').append(foodHTML)
     })
-})
\ No newline at end of file
+})
+
+$submit.on('click', () => {
+  Food.addFood()
+    .then((foodHTML) => {
+      clearInputs()
+      $('#foods-table').append(foodHTML)
```

Comment:
If you wanted the new food at the top of the table, couldn't you just use `prepend` instead of `append`?